CoreScriptsRoot/Modules/Panel3D.lua

```diff
@@ -85,13 +88,20 @@ function Panel3D.Get(panelId)
 
 		panel.transparencyCallbacks = {}
 
+		panel.OnMouseEnter = false
+		panel.OnMouseLeave = false
```

Comment:
We don't do that here? It's functionally similar to nil, but the key will be actually set on the table. Setting it to nil does nothing. Both methods will denote in the code that something is there, so the only difference is in using pairs().